src/fcu_io.cpp

```diff
@@ -127,6 +141,12 @@ void fcuIO::handle_small_imu_msg(const mavlink_message_t &msg)
   sensor_msgs::Temperature temp_msg;
   temp_msg.header.stamp = imu_msg.header.stamp;
 
+  // This is so we can eventually make calibrating the IMU an external service
+  if (!imu_.calibrated)
```

Comment:
Yep, an external service call would be great. We won't want to calibrate every time we start up, but I assume this is just a placeholder for now so that's great